apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -23,6 +23,11 @@ const TitleTagCodec = t.keyof({
   p: null,
 });
 
+const TitleTagH1H2Codec = t.keyof({
```

Comment:
```suggestion
const TitleTagCodec = t.keyof({
```
Simplify variable name and make it more flexible for possible future expansions to allow values other than H1 and H2